dbt_worker/tasks.py

```diff
@@ -101,9 +101,13 @@ def _invoke_runner(
     # after a deps is called. Once core completes the following ticket, we can remove
     # this chdir hack: https://github.com/dbt-labs/dbt-core/issues/6985
     original_wd = os.getcwd()
+    root_path = get_root_path(None, project_dir)
```

Comment:
Can you remind me why we don't use the project_dir, instead we call get_root_path? I feel like get_root_path will return project_dir directly.

Reply:
Or you wanna to mix the semantic use case here as well?

Reply:
@dichenqiandbt Now that we know we won't be using this logic for the SL, I can remove this. Thanks for the catch!